column/__init__.py

```diff
@@ -18,5 +19,12 @@
 
 __version__ = '0.3.6'
 
-cfg = ConfigParser.ConfigParser(allow_no_value=True)
+defaults = {
+    'log_file': os.path.join('/var/log', 'column.log'),
```

Comment:
Also you should have / at all in the strings. Use:

os.path.join('var', 'log', 'column.log')